src/schemathesis/specs/openapi/examples.py

```diff
@@ -38,21 +83,33 @@ def get_static_parameters_from_example(endpoint: Endpoint) -> Dict[str, Any]:
 
 
 def get_static_parameters_from_examples(endpoint: Endpoint, examples_field: str) -> List[Dict[str, Any]]:
+    """Get static parameters from OpenAPI examples keyword."""
     endpoint_def = endpoint.definition.resolved
     return merge_examples(
-        get_param_examples(endpoint_def, examples_field), get_request_body_examples(endpoint_def, examples_field)
+        get_parameter_examples(endpoint_def, examples_field), get_request_body_examples(endpoint_def, examples_field)
     )
 
 
+def get_static_parameters_from_properties(endpoint: Endpoint) -> Dict[str, Any]:
+    endpoint_def = endpoint.definition.resolved
+    return {
+        **get_parameter_example_from_properties(endpoint_def),
+        **get_request_body_example_from_properties(endpoint_def),
+    }
+
+
 def get_strategies_from_examples(endpoint: Endpoint, examples_field: str = "examples") -> List[SearchStrategy[Case]]:
     strategies = [
         get_strategy(endpoint, static_parameters)
         for static_parameters in get_static_parameters_from_examples(endpoint, examples_field)
+        if static_parameters
     ]
-    static_parameters = get_static_parameters_from_example(endpoint)
-    if static_parameters:
-        strategy = get_strategy(endpoint, get_static_parameters_from_example(endpoint))
-        strategies.append(strategy)
+    for static_parameters in static_parameters_union(
+        get_static_parameters_from_example(endpoint), get_static_parameters_from_properties(endpoint)
+    ):
+        strategy = get_strategy(endpoint, static_parameters)
+        if strategy is not None:
```

Comment:
Btw, after recent refactoring, `get_strategy` always returns `SearchStrategy`, so this condition can be removed